python/paddle/__init__.py

```diff
@@ -258,6 +258,7 @@
 from .tensor.math import add  # noqa: F401
 from .tensor.math import subtract  # noqa: F401
 from .tensor.math import logsumexp  # noqa: F401
+from .tensor.math import logaddexp  # noqa: F401
```

Comment:
add `logaddexp` in \_\_all\_\_ list below ?

Reply:
done